### ISSUE TITLE
[BUG] <Problem in the log in page>

### ISSUE THREAD

**Issue by mousamighosh216** [GarimaSingh0109/WasteManagment#189]
### Description

I would like resolve the footer issue as it is covering whole login box. This will enhance the look of the login  page and help user to access website more efficiently.

I would like to work on this issue.
Please assign it to me under gssoc-ext'24.

### Steps to Reproduce

 1. go to main page 
 2. click on login
 3. error is visible i.e. footer is covering login box

### Expected Behavior

Login box should have been visible and footer should have been at the bottom

### Actual Behavior

footer is covering whole login page

### Additional Information

NA

### Browser

Chrome

### Checklist

- [X] I have checked the existing [issues](https://github.com/GarimaSingh0109/WasteManagment/issues).
- [X] I have read the [Code Of Conduct](https://github.com/GarimaSingh0109/WasteManagment?tab=coc-ov-file)
- [X] The changes don't break the code
- [X] Please attach screenshots and videos of the development work once done

**Comment by Anujyadav0074** [comment]
How cna this be solved from here i believe this is a problem on the gsoc website and will be fixed from there end

**Comment by mousamighosh216** [comment]
> How cna this be solved from here i believe this is a problem on the gsoc website and will be fixed from there end

yeah, i was just letting them know. Actually I am new in gssoc didn't knew how to delete this bug as i found an error on resume builder i addressed it but then noticed that website should have been a waste management website so edited that to this one.

**Comment by Anujyadav0074** [comment]
Could we connect on some social account

**Comment by mousamighosh216** [comment]
> Could we connect on some social account

https://www.reddit.com/user/rhythm_lapata/
my reddit account
